src/globalsearch/radiobrowsersearchprovider.cpp

```diff
@@ -0,0 +1,52 @@
+/* This file is part of Clementine.
+   Copyright 2011, David Sansome <me@davidsansome.com>
```

Comment:
For the new files, you should update the copyright date and set yourself as the author.

Reply:
Sorry, I didn't read the full description and see that this is a RFC.